app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -0,0 +1,129 @@
+    question in CSV format, then importing the CSV file to populate the new group.
+
+1. [Add users to the application](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-assign-apps.htm).
+
+1. [Test ID token claims](https://developer.okta.com/docs/guides/customize-authz-server/main/#create-claims) and find groups for mapping. Configure the following claims settings:
+    * **OAuth/OIDC client**: Enter the client name you previously created for your Okta application
+    * **Grant Type**: Authorization Code
+    * **User**: Select an Okta user that is assigned to the {{site.konnect_short_name}} application to test the claim with
+    * **Scope**: `openid`, `email`, `profile`
+
+    In the generated **Preview Token** preview, ensure that the `groups` value is present. From the list of groups in the preview, identify groups that you want to use in {{site.konnect_short_name}}. Take note of these groups.
+
+{% endnavtab %}
+{% navtab Auth0 %}
+Kong offers OIDC support to allow Single-Sign-on for {{site.konnect_short_name}} and the Dev Portal. This guide shows you how to configure Auth0 for Dev Portal SSO.
+
+1. [Create an application in Auth0](https://auth0.com/docs/get-started/auth0-overview/create-applications/machine-to-machine-apps) and configure the following settings:
+    * **Application Type**: Machine to Machine Applications
+    * **Authorized API**: Auth0 Management API
+    * **Permissions**: Authorize the application for at least one permission, for example: `read:client_grants`. 
+    This permission can be revoked later if desired, but one permission is required to create the application.
+
+1. [Configure the Auth0 application settings](https://auth0.com/docs/get-started/applications/application-settings):
+    * **Application Login URI**: `https://cloud.konghq.com/login`
+    * **Allowed Callback URLs**: `https://cloud.konghq.com/login`
+    * **Grant Types**: Authorization Code
+
+1. [Create a login action in Auth0](https://auth0.com/docs/customize/actions/write-your-first-action#create-an-action) and enter the following in the Actions Code Editor:
+    
+    ```js
+    exports.onExecutePostLogin = async (event, api) => {
+    if (event.authorization) {
+        // This transforms the ISO 8601 Timestamp string into the seconds integer representation that is expected for the OIDC standard,
+        // allowing the Konnect SSO validation to accept the format of the `updated_at` property when parsing the token claim.
+        api.idToken.setCustomClaim('updated_at', Math.floor(new Date(event.user.updated_at).getTime()/1000))
+    }
+    };
+    ```
+
+   {:.note}
+   > **Important:** This section is required because the Auth0 API implementation isn't inline with the OIDC standard for the `updated_at` token claim value.
+
+1. Deploy the action by dragging it from the Start to Complete step in the [Login Flow](https://auth0.com/docs/customize/actions/flows-and-triggers/login-flow).
```

Comment:
Should it be this link?
https://auth0.com/docs/customize/actions/write-your-first-action#deploy-the-action